src/App.jsx

```diff
@@ -14,70 +15,70 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const handleClick = good => setSelectedGood(good);
+  const resetClick = () => setSelectedGood('');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood ? `${selectedGood} is selected` : 'No goods selected'}
+        {selectedGood && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={resetClick}
+          />
+        )}
+      </h1>
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+      <table className="table">
+        <tbody>
+          {goods.map((product) => {
+            const isSelected = selectedGood === product;
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+            return (
+              <tr
+                data-cy="Good"
+                key={product}
+                className={isSelected ? (
+                  'has-background-success-light'
+                ) : (
+                  undefined
+                )}
```

Comment:
Consider it will be better
```suggestion
                className={isSelected ? 'has-background-success-light' :  ''}
```